DependencyInjection/Configuration.php

```diff
@@ -77,10 +77,11 @@ public function getConfigTreeBuilder()
                     ->prototype('array')
                         ->children()
                             ->scalarNode('locale')->defaultValue('default')->end()
-                            ->booleanNode('xhtml')->defaultValue(true)->end()
-                            ->booleanNode('strict')->defaultValue(true)->end()
-                            ->booleanNode('escaping')->defaultValue(true)->end()
-                            ->booleanNode('line_breaks')->defaultValue(true)->end()
+                            ->booleanNode('xhtml')->defaultTrue()->end()
+                            ->booleanNode('strict')->defaultTrue()->end()
+                            ->booleanNode('escaping')->defaultTrue())->end()
+                            ->booleanNode('line_breaks')->defaultTrue())->end()
```

Comment:
Agreed, but i think boolean nodes should have default*() setters. This changes doesn't relate to newline setting.